lib/shoryuken/core_ext.rb

```
@@ -14,7 +14,7 @@ def stringify_keys
     module SymbolizeKeys
       def symbolize_keys
         keys.each do |key|
-          self[(key.to_sym rescue key) || key] = delete(key)
+          self[key.to_sym rescue key || key] = delete(key)
```

RuboCop (1.76.1)

[fatal] expected a matching `]`
(Using Ruby 3.4 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)